src/schemas/xspec.rnc

```diff
@@ -196,7 +196,11 @@ global-param =
 	## Global parameters are set globally, for the entire unit test suite. There's
 	## no way to set global parameters within a particular scenario (due to
 	## constraints on implementation).
-	element param { common-attributes, name, as?, selection }
+	element param { common-attributes, name, as?, static?, selection }
+
+static = 
+	## Enabling this attribute is supported only when /x:description/@run-as is "external".
```

Comment:
> 1. It would be useful to check this condition in `xspec.sch` (if not now, maybe later).

It is legal to write `/x:description[empty(@schematron|@query|@stylsheet)]` as long as it is imported (directly or indirectly) into `/x:description[exists(@schematron|@query|@stylsheet)]`. For example, [`test/line-number/common.xspec`](https://github.com/xspec/xspec/blob/master/test/line-number/common.xspec) imported by [`test/line-number_disabled.xspec`](https://github.com/xspec/xspec/blob/master/test/line-number_disabled.xspec) has no attributes in `/x:description`.

Likewise, I think `/x:description[empty(@run-as)]/x:param[@static = 'yes']` should be allowed for convenience.

> Maybe also check that static parameters don't appear in tests for XQuery or Schematron, unless you think it's better to silently ignore the attribute in those cases.

Committed 6ef1f27bd3b0076b250e4af8869389aba15ff228 to reject static parameters on XQuery and Schematron.

> 2. Should this attribute be labeled as experimental because of its dependency on the experimental `run-as="external"` feature?

[Fixed](https://github.com/xspec/xspec/commit/6ef1f27bd3b0076b250e4af8869389aba15ff228#diff-3d6b4de8d205258c1cd8b0997cea031d)

Reply:
Thanks. I hadn't thought about the case of importing an XSpec file, but you make a good point.
The updates you've made today all look good.